templates/solutions/secure-open-source/index.html

```diff
@@ -509,6 +498,17 @@ <h3 class="p-heading--5">Manage, monitor, secure and audit with Landscape</h3>
           <h3 class="p-heading--5">Patch your systems automatically and without downtime with Livepatch</h3>
         </div>
         <div class="col-6 col-medium-3">
+          <div class="p-media-container u-aspect-ratio--16-9 u-background--fade u-align--center u-vertically-center">
+            {{ image (
+              url="https://assets.ubuntu.com/v1/498e1472-Livepatching+at-a-glance-diagram-1.png",
+              alt="Livepatching automation diagram",
+              width="1881",
+              height="1185",
```

Comment:
I think width/height need to be adjusted

Reply:
Adjusted to actual width of the image.